### PR TITLE
Updating README for starting the container

### PR DESCRIPTION
**Pull request by rajasec**
README to include the latest changes for running the container
Signed-off-by: rajasec <rajasec79@gmail.com>

### CODE REVIEW

**Comment by hqhq** [comment]
One more spot needs to be changed, the service script.

**Comment by rajasec** [comment]
@hqhq 
updated the service script.

**Comment by hqhq** [comment]
LGTM

**Comment by cyphar** [comment]
Can you also update `main.go`, `spec.go` and `tests/integration/*`?

**Comment by rajasec** [comment]
@cyphar 
Updated the files.

File: tests/integration/README.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ function teardown() {
 }
 
 @test "this is a simple test" {
-  runc start containerid
+  runc run containerid
   # "run" automatically populates $status, $output and $lines.
```
Comment (cyphar) [inline tests/integration/README.md]: I just realised that I didn't update this comment. Just say "the runc macro" instead of "run".

**Comment by rajasec** [comment]
@cyphar 
Updated. Sorry I did not noticed the comment.

**Comment by cyphar** [comment]
LGTM

[Approved with PullApprove](https://pullapprove.com/opencontainers/runc/pull-request/877/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=opencontainers/runc)

**Comment by hqhq** [comment]
LGTM

**Pull request merged**